money-core/src/main/scala/com/comcast/money/emitters/LogEmitter.scala

```diff
@@ -38,34 +35,48 @@ object LogEmitter {
       case _ => Props(classOf[LogEmitter], conf)
     }.get
 
-  def buildMessage(t: Span): String = {
+  def buildMessage(t: Span)(implicit conf: Config): String = {
     implicit val builder = new StringBuilder()
-    builder.append("Span: ")
-    append("span-id", t.spanId.spanId)
-    append("trace-id", t.spanId.traceId)
-    append("parent-id", t.spanId.parentSpanId)
-    append("span-name", t.spanName)
-    append("app-name", Money.applicationName)
-    append("start-time", t.startTime)
-    append("span-duration", t.duration)
-    append("span-success", t.success)
+    builder.append(configValue("span-start", "Span: "))
```

Comment:
A few comments on this...

It would seem to make sense to parse the Config into some type and use that as a val first.  

Rather than having a method on LogEmitter that gets pulled in, just have a class.  Some psuedo code:

```scala
object LogFormatter {

  // by providing a default setup, you don't have to check has path every time you log
  val default =
    """
      |keys {
      |   span-id = "span-id"
      |   trace-id = "trace-id"
      |   parent-id = "parent-id"
      |   span-name = "span-name"
      |   app-name = "app-name"
      |   start-time = "start-time"
      |   span-duration = "span-duration"
      |   span-success = "span-success"
      |}
    """.stripMargin

  val defaultConfig = ConfigFactory.parseString(default)

  def apply(conf: Config): LogFormatter = {
    // the passed in config with fallback will ensure there are default values
    val config = conf.withFallback(defaultConfig)

    // build the log formatter with the config values.  You can create this once when the actor is created...
    new LogFormatter(config.getString("keys.span-id"), config.)
  }
}

/* By having the loging done here, you don't have to inspect the config when logging */
class LogFormatter(val spanId: String, val traceId: String, val parentId: String, val spanName: String,
                   val appName: String, val startTime: String, val spanDuration: String, val spanSuccess: String) {

  def format(span: Span): String = ??? // TODO: fun-times here
}
```